capstone-frontend/app.yaml

```diff
@@ -1,5 +1,5 @@
 runtime: nodejs12
-service: reactapp # Name of Service, instead of default
+service: default
```

Comment:
I saw you changed this to "reactapp" on the last PR, and then changed it back to "default" on this one. Just wanted to make sure this was your intention.